service/build.gradle

```diff
@@ -73,3 +76,18 @@ tasks.register("pactTests", Test) {
 		includeTags "pact-test"
 	}
 }
+
+tasks.register("verifyPacts", Test) {
+	useJUnitPlatform {
```

Comment:
Out of curiosity, why the crazy indenting?

Reply:
It's the same as the rest of the file
Although I _did_ just realizes that this file uses tabs instead of spaces. eww....